commerce/domain/pricing/service/classes/PricingServiceSample.cls

```diff
@@ -157,10 +203,9 @@ public class PricingServiceSample extends commercestorepricing.PricingService {
     }
 
     // To access the service below, you may need to add endpoint =
-    // https://b2b-commerce-test.herokuapp.com in Setup > Security > Remote site settings.
+    // httpHost in Setup > Security > Remote site settings.
     String requestURL =
-      'https://b2b-commerce-test.herokuapp.com/get-prices?skus=' +
-      JSON.serialize(encodedProductIDs);
+      httpHost+'/get-prices?skus=' +  JSON.serialize(encodedProductIDs);
```

Comment:
I don't understand this portion. Can you add some comments?

Reply:
This is where the heroku host is changed to the httpHost, so that we no longer make calls to the Heroku host